<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrarMesa</class>
 <widget class="QDialog" name="RegistrarMesa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>atras</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_helados">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Helados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cocteles">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cocteles</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_platos">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Platos de Fondo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_guard">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Guarniciones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_entrada">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Entradas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_postres">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Postres</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sand">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sandwichs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_segund">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Segundos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sopas">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sopas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calientes">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Bebidas calientes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fria">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Bebidas fria</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_combo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Combos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_jugo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Jugos</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_productos">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>561</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Precio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Stock</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_muestra">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>561</width>
     <height>201</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nombres</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Precio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cantidad</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sopas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_aumentar">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reducir">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quitar">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>quitar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_escojer">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Escojer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_puesto">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_saldo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>520</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>descuento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>total</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_registrar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Pasar a Caja</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
